pandas/core/frame.py

```diff
@@ -2735,6 +2736,81 @@ def to_parquet(
             **kwargs,
         )
 
+    def to_orc(
+        self,
+        path: FilePathOrBuffer = None,
+        engine: str = 'pyarrow',
```

Comment:
Could use Literal

`engine: Literal["pyarrow", "other_engine"] = 'pyarrow',`

Reply:
I assume you meant `Literal["pyarrow"]` .

Reply:
Since it can also be a Module, probably `Literal["pyarrow"] | ModuleType`. I'm not sure whether we want `ModuleType` since we will need ignores for each function of the module.

Reply:
Thanks! Since using a module itself as the `engine` argument is not permitted in `to_parquet` I see no reason to allow it in `to_orc`.